src/main.cc

```diff
@@ -288,11 +288,212 @@ NAN_METHOD(EnumKeys) {
     RegCloseKey(hCurrentKey);
 }
 
-NAN_MODULE_INIT(Init) {
-  Nan::SetMethod(target, "readValues", ReadValues);
-  Nan::SetMethod(target, "enumKeys", EnumKeys);
-}
+  NAN_METHOD(CreateKey)
+  {
+    auto argCount = info.Length();
+    if (argCount != 1 && argCount != 2)
```

Comment:
Shouldn't `argCount` always be 2? What's the use case for 1?

The code below does basically nothing when there is only 1 argument, and it even returns `true` as if the function succeeded 🤔

Reply:
you're right